com.reprezen.swagedit.openapi3/resources/templates.xml

```diff
@@ -0,0 +1,328 @@
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+
+     <template name="put" 
+       id="com.reprezen.swagedit.openapi3.templates.path_put_template" 
+       description="PUT template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">put:
+      summary: Update
+      description: Update the entire list (for collection resources) or an item (for object resources)
+      parameters:
+        - name: ${request}
+          in: body
+          required: true
+          schema:
+            $$ref: ${model}  
+      responses:
+        '${200}':
+          description: OK</template>
+
+     <template name="delete" 
+       id="com.reprezen.swagedit.openapi3.templates.path_delete_template" 
+       description="DELETE template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">delete:
+      summary: Delete 
+      description: Delete the entire list (for collection resources) or an item (for object resources)
+      responses:
+        '${200}':
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+    
+    <template name="patch" 
+       id="com.reprezen.swagedit.openapi3.templates.path_patch_template" 
+       description="PATCH template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">patch:
+      summary: Partial update
+      description: ${Update}
+      parameters:
+        - name: ${request}
+          in: body
+          required: true
+          schema:
+           $$ref: ${model}  
+      responses:
+        '${200}':
+          description: OK
+          schema:
+            $$ref: ${model}</template>
+
+    <template name="options" 
+       id="com.reprezen.swagedit.openapi3.templates.path_options_template" 
+       description="OPTIONS template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">options:
+      summary: Request resource options
+      description: Request information about methods available for resource manipulation 
+      responses:
+        '${200}':
+          description: OK</template>
+
+    <template name="head" 
+       id="com.reprezen.swagedit.openapi3.templates.path_head_template" 
+       description="HEAD template" 
+       context="com.reprezen.swagedit.openapi3.templates.path_item" 
+       enabled="true">head:
+      summary: Query a resource in the server
+      description: HEAD sends request and get the response only in the header, the body should not be used
+      responses:
+        '${200}':
+          description: OK</template>
+          
+<!--
+	RESPONSES
+ -->
+ 
+	<template name="OK response" 
+       id="com.reprezen.swagedit.openapi3.templates.200_response" 
+       description="OK template" 
+       context="com.reprezen.swagedit.openapi3.templates.responses" 
+       enabled="true">2${00}:
+      description: ${OK}</template>
```

Comment:
Indentation is broken here: it's 6 spaces, while it should be 10.
We use `com.reprezen.swagedit.templates.SwaggerTemplateContext.IndentationAwareTemplateTranslator` to make templates indentation-aware for Swagger v2. We can move this class the core plugin and reuse it for OpenAPI v3.